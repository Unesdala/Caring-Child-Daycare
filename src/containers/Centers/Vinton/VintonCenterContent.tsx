const VintonCenterContent = (): JSX.Element => (
  <main className="vinton-main--content">
    <section className="vinton-main--introduction">
    <div className="introduction-grid">
        <div className="location-features">
          <h1>
            Location Features
          </h1>
          <ol className="centers-list">
            <li>Secure Entrances</li>
            <li>Spacious rooms filled with colourful patterns to stimulate learning</li>
            <li>Music rooms for kids to explore their creative talents</li>
            <li>Modern design with skylight for natural lighting and a view of the sky</li>
            <li>Food menus designed by our in-house chefs, designed to meet USDA guidelines</li>
            <li>A large playground for the children to play</li>
            <li>Off site Field Trips</li>
            <li>Free parking for drop off and pick up</li>
          </ol>
        </div>
        <div className="location-directions">

        </div>
      </div>
    </section>
    <section className="vinton-main--staff">
      <h1 className="staff-header">Meet Our Vinton Director Team</h1>
      <div className="staff-grid">
        <div className="staff-grid--director">
          <div className="staff-grid--picture">
            <img src="https://dl.dropboxusercontent.com/s/o6xtpfqxibbubh3/Placeholder.png?dl=0" alt="Placeholder Image" />
          </div>
          <div className="staff-grid--name"><h2>Candi Sweet</h2></div>
          <div className="staff-grid--title"><h2>Director</h2></div>
          <div className="staff-grid--description">
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Rem amet totam aliquid, optio facere 
              necessitatibus a mollitia consequatur doloribus voluptates quos debitis repellat. Molestias iure dicta ab et aut.
            </p>
            <p>
              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Dolorum esse suscipit ad eos nobis! 
              Ipsam, cupiditate deserunt, beatae inventore odit officia corrupti possimus nemo, culpa fugit ipsa quam 
              eum vel minus praesentium eveniet voluptas dicta ea fuga.
            </p>
            <p>
              Lorem, ipsum dolor sit amet 
              consectetur adipisicing elit. At impedit asperiores quasi, animi est itaque reprehenderit repellendus nemo eum incidunt nam tempora.
            </p>
          </div>
        </div>
        <div className="staff-grid--assistant-director">
          <div className="staff-grid--picture">
            <img src="https://dl.dropboxusercontent.com/s/o6xtpfqxibbubh3/Placeholder.png?dl=0" alt="Placeholder Image" />
          </div>
          <div className="staff-grid--name"><h2>Name</h2></div>
          <div className="staff-grid--title"><h2>Assistant Director</h2></div>
          <div className="staff-grid--description">
            <p>
              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nisi blanditiis iste tempora, aut facere expedita 
              molestiae omnis aspernatur suscipit, accusantium corporis fugiat voluptas sapiente ad repellat quos rerum dolorum corrupti?
            </p>
            <p>
              Lorem ipsum dolor sit amet consectetur, adipisicing elit. Error neque aperiam rem quas similique 
              totam voluptatem molestiae assumenda eaque et, aut, consectetur dolores?
            </p>
          </div>
        </div>
        <div className="staff-grid--third-in-charge">
          <div className="staff-grid--picture">
            <img src="https://dl.dropboxusercontent.com/s/o6xtpfqxibbubh3/Placeholder.png?dl=0" alt="Placeholder Image" />
          </div>
          <div className="staff-grid--name"><h2>Name</h2></div>
          <div className="staff-grid--title"><h2>Third in Charge</h2></div>
          <div className="staff-grid--description">
            <p>
              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Accusamus magni error at earum excepturi 
              inventore debitis officia dolore soluta odio velit, ipsa iste. Totam blanditiis exercitationem incidunt at adipisci?
            </p>
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam molestias sequi assumenda dolores enim 
              cumque labore culpa est ut! Velit ipsum porro, totam fugiat atque placeat facere iure consequuntur adipisci qui quasi tempore in minima.
            </p>
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequatur sequi fuga alias quasi soluta neque, porro dolores earum maxime.
            </p>
          </div>
        </div>
      </div>
    </section>
  </main>
);

export default VintonCenterContent;
