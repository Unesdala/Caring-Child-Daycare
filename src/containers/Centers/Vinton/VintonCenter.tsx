import VintonCenterContent from './VintonCenterContent';

export const VintonCenter = (): JSX.Element => {
  document.title = 'Caring Child Daycare | Vinton Center';  
  return (
    <div className="Site-content">
      <main className="vinton-main">
        <VintonCenterContent />
      </main>
    </div>
  );
};

export default VintonCenter;
