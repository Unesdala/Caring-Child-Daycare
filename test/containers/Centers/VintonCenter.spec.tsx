/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { VintonCenter } from 'src/containers/Centers/Vinton/VintonCenter';
import VintonCenterContent from 'src/containers/Centers/Vinton/VintonCenterContent';

describe('Vinton Center', () => {
  const wrapper = shallow(<VintonCenter />);
  it('renders', () => {
    expect(wrapper.find(VintonCenterContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(VintonCenterContent).dive().find('main.vinton-main--content').exists()).toBe(true);
  });
  
});
