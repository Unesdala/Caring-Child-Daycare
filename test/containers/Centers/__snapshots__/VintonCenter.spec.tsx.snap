// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vinton Center renders 1`] = `ShallowWrapper {}`;
